jekyll/_cci2/release/configure-your-kubernetes-components.adoc

```diff
@@ -0,0 +1,223 @@
+        version: 1.0.0
+----
+
+Once you have updated your Deployment or Rollout, check the CircleCI releases dashboard and you should see your release in the timeline view.
+
+#screenshot#
+
+[#link-release]
+== 2. Link release to deployment job trigger (optional)
+
+You can link your release with the CircleCI _deployment_ job that triggered it. To do this, add the following annotations to your Deployment/Rollout. The method for incorporating these values and variables will depend on you config and tooling:
+
+* `circleci.com/job-number` should be set to the `CIRCLE_BUILD_NUM` xref:../variables#built-in-environment-variables[environment variable].
+* `circleci.com/pipeline-id` should be set to the `pipeline.id` xref:../variables#pipeline-values[pipeline value].
+* `circleci.com/workflow-id` should be set to the `CIRCLE_WORKFLOW_ID` xref:../variables#built-in-environment-variables[environment variable].
+
+Once these values and variables are rendered, the config will look something like:
+
+[,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    circleci.com/job-number: "1"
+    circleci.com/pipeline-id: 88dfee99-0348-407f-b113-dbf270cad093
+    circleci.com/workflow-id: 5b8c4de8-fd5f-4be2-80a4-3d0c03fc138c
+----
+
+[#configure-release-management]
+== 3. Configure release management (optional)
+
+By adding a few additional annotations to your Kubernetes objects (Deployment/Rollout), you can enable additional actions on your releases dashboard. This includes restoring, scaling, and restarting component versions. To get started, add the `circleci.com/project-id` annotation with the value found on the relevant CircleCI project settings page.
+
+[,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    ...
+    circleci.com/project-id: 9da0c100-3295-49a4-827f-7892f3e8dc83
+----
+
+#Screenshot#
+
+[#helm-rollback]
+=== a. Use Helm rollback
+
+CAUTION: This option is only available when using Helm to configure your Kubernetes resources.
+
+By default the built-in logic for Kubernetes Deployments or Argo Rollouts is used for the restore version feature. If you manage your component with Helm, you can choose to use the Helm rollback strategy instead. To do so, add the `circleci.com/helm-revision-number` annotation to the Kubernetes object metadata in your Helm chart template:
+
+{% raw %}
+[,yaml]
+----
+annotations:
+  circleci.com/helm-revision-number: {{ .Release.Revision | quote }}
+----
+{% endraw %}
+
+[#operation-timeout]
+=== b. Custom operation timeout
+
+CAUTION: This option is only available when using Helm to configure your Kubernetes resources.
+
+The `circleci.com/operation-timeout` annotation allows a custom timeout to be specified for Helm Rollback operations performed as part of a Restore Version command. Valid values are Go duration strings (for example, 5m, 10m15s). This option is available if you are using Helm to manage your Kubernetes resources.
```

Comment:
~ Would it be worth linking to `https://pkg.go.dev/maze.io/x/duration#Duration` here?